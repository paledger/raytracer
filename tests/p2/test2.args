pixelcolor ../resources/simple.pov 640 480 360 195
