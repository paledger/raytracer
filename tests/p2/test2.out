Pixel: [360, 195] Ray: {0 0 14} -> {0.08372 -0.09199 -0.9922}
T = 12.91
Object Type: Sphere
BRDF: Blinn-Phong
Color: (51, 0, 51)
